class_project_instructions/README.md

```diff
@@ -160,13 +160,13 @@
 
 - **Add Files Only in Your Project Directory**
   - Add your project files under the following directory:
-    `{GIT_ROOT}/DATA605/Spring2025/projects/{branch_name}`
+    `{GIT_ROOT}/COURSECODE/Term20xx/projects/{branch_name}`
```

Comment:
Let's encode it as

`{GIT_ROOT}/{COURSE_CODE}/{TERM}{YEAR}`

E.g., `{GIT_ROOT}/DATA605/Spring2025` or `{GIT_ROOT}/MSML610/Fall2026`